python/src/main/java/org/apache/zeppelin/python/IPythonInterpreter.java

```diff
@@ -137,7 +138,10 @@ public void open() throws InterpreterException {
       int jvmGatewayPort = RemoteInterpreterUtils.findRandomAvailablePortOnAllLocalInterfaces();
       LOGGER.info("Launching IPython Kernel at port: " + ipythonPort);
       LOGGER.info("Launching JVM Gateway at port: " + jvmGatewayPort);
-      ipythonClient = new IPythonClient("127.0.0.1", ipythonPort);
+      int framesize = Integer.parseInt(getProperty("zeppelin.ipython.grpc.framesize",
+          4 * 1024 * 1024 + ""));
+      ipythonClient = new IPythonClient(ManagedChannelBuilder.forAddress("127.0.0.1", ipythonPort)
+          .usePlaintext(true).maxInboundMessageSize(framesize));
```

Comment:
does this `.usePlaintext(true)` change the behavior?

Reply:
No, it is the same as previous behavior.